docs/docs/cli-commands.md

```diff
@@ -2279,10 +2279,12 @@ yarn redwood upgrade -t 0.19.3
 
 The CLI can check for things in the background, like new versions of the framework, while you dev.
 
-Right now it can only check for new versions.
-If you'd like it to do so, set `notifications.versionUpdates` in the `redwood.toml` file to include an array of the tags you're interested in hearing about.
-(The former has priority.)
+Right now it defaults to check for a new `latest` version.
```

Comment:
```suggestion
Right now it defaults to check for a new "latest" version.
```

Reply:
I think "checking" is the more correct form of the verb here 🙂